tests/tests.sh

```diff
@@ -1,4 +1,22 @@
 set -e
 
-sssom parse --input /Users/matentzn/ws/sssom-py/tests/data/basic.tsv --output /Users/matentzn/ws/sssom-py/tests/tmp/basic.tsv.tsv --input-format tsv --curie-map-mode merged
```

Comment:
It's possible to write proper unit tests for CLIs using the CliRunner in click
https://click.palletsprojects.com/en/8.0.x/testing/

but it's better to write tests for the functions the CLI wraps

Reply:
Oh we have that as well - I just think we want both :) 
https://github.com/mapping-commons/sssom-py/blob/cli_edit/tests/test_cli.py

Reply:
We also have tests for the functions that the CLI wraps: 
https://github.com/mapping-commons/sssom-py/blob/cli_edit/tests/test_conversion.py